src/screens/Register/Register.jsx

```diff
@@ -55,6 +67,14 @@ const Register = ({ image }) => {
     history.push("/login");
   }, [history]);
 
+  const handleReadPrivacy = useCallback(() => {
+    setReadPrivacy(!readPrivacy);
+  }, [readPrivacy]);
```

Comment:
change to this, so that the useCallback dependency is not necessary.
```suggestion
    setReadPrivacy(readPrivacy => !readPrivacy);
  }, []);
```